README.md

```diff
@@ -55,29 +55,40 @@ gradlew sonarQube -DsonarRunner.aemVersion=6.4
 
 Below you will find descriptions of all rules available in **AEM Rules for SonarQube** plugin.
 
-## Good practices
+## AEM Good practices
 
 - **AEM-1** Use predefined constant in annotation instead of hardcoded value.
   - Use constants available in AEM instead of repeating inline literals.
 
 - **AEM-2** Use predefined constant instead of hardcoded value.
   - Use constants available in AEM instead of repeating inline literals.
+  
+- **AEM-4** Implicit search strategy used in Sling Query
+  - `SearchStrategy` can have negative performance impact if mismatched.
+  Therefore developer should always make informed decision and define strategy explicitly.
+  
+- **AEM-5** ``getContentResource()`` is not null checked
+  - Always null check the return value of ``getContentResource()``. It is possible to get a null if a jcr:content node does not exist in the repository.
 
 - **AEM-8** Prefer cleaner `@SlingServlet` annotation.
   - Prefer cleaner `@SlingServlet` annotation over `@Properties` approach. Do not mix up both approaches.
 
-- **AEM-14** Using http literal hardcoded makes it difficult to switch to https later on.
+- **AEM-9** No mutator methods invoked on ``ModifiableValueMap``
+  - ``ModifiableValueMap`` should be replaced by ``ValueMap`` if no mutator methods are invoked.
+
+- **AEM-12** Using http literal hardcoded makes it difficult to switch to https later on.
```

Comment:
Arguably, this has nothing to do with AEM.

Reply:
This rule is no longer maintained:
https://github.com/Cognifide/AEM-Rules-for-SonarQube/pull/146/files#diff-04c6e90faac2675aa89e2176d2eec7d8